src/state.rs

```diff
@@ -446,6 +446,7 @@ mod tests {
         let mut world = World::new();
 
         let mut sm = StateMachine::new(State1(7));
+        // Unwrap here is fine because start can only fail when there are no states in the machine.
```

Comment:
This is a test :sweat_smile: no need to justify unwraps here ;)

Reply:
Eh I already wrote it, not much harm in leaving it right?

Reply:
No :)